Mathlib/Analysis/InnerProductSpace/Dual.lean

```diff
@@ -45,8 +45,10 @@ namespace InnerProductSpace
 
 open RCLike ContinuousLinearMap
 
+section Seminormed
+
 variable (𝕜 : Type*)
-variable (E : Type*) [RCLike 𝕜] [NormedAddCommGroup E] [InnerProductSpace 𝕜 E]
+variable (E : Type*) [RCLike 𝕜] [SeminormedAddCommGroup E] [InnerProductSpace 𝕜 E]
```

Comment:
Can you move
```
variable (𝕜 : Type*) (E : Type*)
```
outside the `section` so that you don't need to repeat it?

Reply:
done. Is it ok to have `[RCLike 𝕜]` inside `section`?